src/huggingface_hub/repocard.py

```diff
@@ -146,10 +150,25 @@ def metadata_update(
             attempting to overwrite an existing field will cause an error.
         token (`str`, *optional*):
             The Hugging Face authentication token.
-
+        commit_message (`str`, *optional*):
+            The summary / title / first line of the generated commit. Defaults to
+            `f"Update metdata with huggingface_hub"`
+        commit_description (`str` *optional*)
+            The description of the generated commit
+        revision (`str`, *optional*):
+            The git revision to commit from. Defaults to the head of the
+            `"main"` branch.
+        create_pr (`boolean`, *optional*):
```

Comment:
What happens when I pass `create_pr=True` with `revision=None`?

Reply:
It creates a PR against the `main` branch